README.md

```diff
@@ -14,4 +14,4 @@ Type "help", "copyright", "credits" or "license" for more information.
 >>> keyring.set_password("relish-notifier", "EMAIL", "<your email>")
 >>> keyring.set_password("relish-notifier", "PASSWORD", "<your password>")
 >>> exit()
-`
\ No newline at end of file
+`
```

Comment:
Can you actually make this three ` chars?

Reply:
done